src/bin/ion/commands/mod.rs

```diff
@@ -155,12 +155,18 @@ impl WithIonCliArgument for ClapCommand {
         )
     }
 
-    fn with_compression_control(self) -> Self {
```

Comment:
🗺️ Prior to this change, `ion help` would include the `--no-auto-decompress` flag in its output. This was potentially confusing, as `ion help` (and `ion` itself) take no input.